test/MWD.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2016 Signal K and contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const Parser = require('../lib')
+const chai = require('chai')
+const should = chai.Should()
+
+chai.use(require('chai-things'))
+
+describe('MWD', () => {
+
+  it('speed & direction data', () => {
+      const delta = new Parser().parse('$IIMWD,,,046.,M,10.1,N,05.2,M*0B')
+
+    delta.updates[0].values.should.contain.an.item.with.property('path', 'environment.wind.speedTrue')
+    delta.updates[0].values[1].value.should.be.closeTo(5.19585, 0.00005)
```

Comment:
The test data contains 5.2 meters per second, so why do we convert from knots? Which one is more correct?

Reply:
Got a point there. I changed the implementation to give preference to the m/s value and changed the test so equality is used when possible and closeTo when the value had to be converted.